### PR TITLE
chore(demo): `InputPhoneInternational` update `API` page

### PR DESCRIPTION
**Pull request by mdlufy**
Partially solved #9057

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/addon-doc/components/api/api-item.style.less [hunk 1]
```diff
@@ -54,6 +54,11 @@
 .t-input {
     min-inline-size: 10rem;
     margin-block-start: -0.625rem;
+
+    code {
+        overflow: hidden;
+        white-space: nowrap;
+    }
```
Comment (mdlufy) [inline projects/addon-doc/components/api/api-item.style.less:61]: Maybe more preferably add directly to https://github.com/taiga-family/taiga-ui/blob/main/projects/addon-doc/components/main/main.style.less#L44-L75

Comment (waterplea) [reply projects/addon-doc/components/api/api-item.style.less:61]: Let's leave it here for now, and if we meet it again – we'll move it.

File: projects/demo/src/modules/components/input-phone-international/index.ts [hunk 2]
```diff
@@ -23,6 +27,10 @@ import {InheritedDocumentation} from '../abstract/inherited-documentation';
         InheritedDocumentation,
         ReactiveFormsModule,
         TuiDemo,
+        TuiDocControl,
+        TuiDocDropdown,
+        TuiDocIcons,
+        TuiDocTextfield,
```
Comment (waterplea) [inline projects/demo/src/modules/components/input-phone-international/index.ts:33]: Please also remove `tuiProvide(ABSTRACT_PROPS_ACCESSOR, PageComponent),` and `extends AbstractExampleTuiControl` from this file

Comment (mdlufy) [reply projects/demo/src/modules/components/input-phone-international/index.ts:25]: Removed

**Review by waterplea — APPROVED** [review]

**Pull request merged**